ofono/.gitignore

```diff
@@ -36,7 +36,6 @@ test-driver
 test-suite.log
 unit/test-common
 unit/test-util
-unit/test-idmap
```

Comment:
Why is that?

Reply:
I guess that test could remain.  Upstream removed it because they no longer use idmap, instead relying on ELL.